collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java

```diff
@@ -67,36 +69,33 @@
            static final String GOOGLE_MAPS_BASEMAP_DEFAULT  = "streets";
            static final String OSM_MAPS_BASEMAP_DEFAULT     = "mapquest_streets";
 
-    /**
-     * General keys
-     */
-    public static final Collection<String> ALL_GENERAL_KEYS = Arrays.asList(
-            KEY_SHOW_SPLASH,
-            KEY_SPLASH_PATH,
-            KEY_FONT_SIZE,
-            KEY_DELETE_AFTER_SEND,
-            KEY_INSTANCE_SYNC,
-            KEY_PROTOCOL,
-            KEY_PROTOCOL_SETTINGS,
-            KEY_SELECTED_GOOGLE_ACCOUNT,
-            KEY_USERNAME,
-            KEY_FORM_METADATA,
-            KEY_METADATA_USERNAME,
-            KEY_METADATA_PHONENUMBER,
-            KEY_METADATA_EMAIL,
-            KEY_SERVER_URL,
-            KEY_GOOGLE_SHEETS_URL,
-            KEY_FORMLIST_URL,
-            KEY_SUBMISSION_URL,
-            KEY_NAVIGATION,
-            KEY_CONSTRAINT_BEHAVIOR,
-            KEY_COMPLETED_DEFAULT,
-            KEY_MAP_SDK,
-            KEY_MAP_BASEMAP,
-            OSM_BASEMAP_KEY,
-            GOOGLE_MAPS_BASEMAP_DEFAULT,
-            OSM_MAPS_BASEMAP_DEFAULT,
-            KEY_AUTOSEND_WIFI,
-            KEY_AUTOSEND_NETWORK
-    );
+    private static HashMap<String, Object> getHashMap() {
```

Comment:
Sorry I didn't get back to you sooner --  I do think language should be included. I also notice analytics is missing.